libs/ngx-sherfire/src/lib/app/sherfire-app.service.ts

```diff
@@ -0,0 +1,16 @@
+import { inject, Injectable, NgZone } from '@angular/core';
+import { initializeApp, type FirebaseApp } from 'firebase/app';
+import { FirebaseConfig } from '../firebase-config.service';
+import { NgxSherfireModule } from '../ngx-sherfire.module';
+
+export interface SherfireApp extends FirebaseApp {}
+
+@Injectable({
+    providedIn: NgxSherfireModule,
+    useFactory: () =>
+        inject(NgZone).runOutsideAngular(() => {
+            const config = inject(FirebaseConfig);
+            return initializeApp(config, config.appSettings);
```

Comment:
Second argument to `initializeApp` is new and contains new setting `automaticDataCollectionEnabled`